README.md

```diff
@@ -1,2 +1,33 @@
-# loopback-sdk-ios-codegen
-Code generation tool for LoopBack iOS SDK
+# LoopBack iOS SDK CodeGen
+
+The iOS CodeGen command line tool generates iOS client side LoopBack Model representations
+in Objective-C by looking into the specified server application.
+
+## How to Use
+
+The following is an example of the usage applied to the test server provided under this directory:
+
+ 1. Run `npm install` in this directory to initialize the tool `bin/lb-ios` and the test server under `test-env/server`.
+ *  Run `bin/lb-ios -p XX test-env/server/server test-env/client/ios/gen-src` 
+ 	to generate the models in Objective-C representation, 
+ 	where `XX` is the prefix attached to all the generated class names.
+
+## How to Run the Unit Test
+
+After performing the above steps, run the followings:
+
+ 1. Run the test server by `node test-env/server/server.js &`.
+ *  Copy `LoopBack.framework` into `test-env/CodeGenTest/`.
```

Comment:
Is there a way how to reference source files from LoopBack.framework directly, so that it is not necessary to copy them?

If not, then I am proposing the create an npm script to run the generator and copy the files in one step.

Intended usage: `npm run test-setup`

Implementation - add a new script entry to package.json:

```json
{
  "scripts": {
    "test-setup": "bin/lb-ios -p XX test-env/server/server test-env/client/io/gen-src && cp -R LoopBack.framework test-env/CodeGenTest"
  }
}
```

Thoughts?